pkg/a8rcloud/config.go

```diff
@@ -0,0 +1,19 @@
+package a8rcloud
+
+import (
+	"crypto/x509"
+	"fmt"
+
+	"github.com/telepresenceio/telepresence/rpc/v2/manager"
+)
+
+func certsFromConfig(cfg *manager.AmbassadorCloudConfig) (*x509.CertPool, error) {
```

Comment:
Shouldn't this function always return at least what's in `x509.SystemCertPool()` and then append the poxyCA if it is present?

Reply:
I'm not super sure it makes a difference insofar as if we're behind a proxy presumably everything will be proxied, but i can go add it it's not hard